pkg/parse/namespace.go

```diff
@@ -88,11 +88,19 @@ func (p *namespace) parseSource(ctx context.Context, state sourceState) ([]ast.F
 		// So disable the API call and dynamic mode of NamespaceSelector.
 		AllowAPICall:             false,
 		DynamicNSSelectorEnabled: false,
+		WebhookEnabled:           p.WebhookEnabled,
 	}
 	options = OptionsForScope(options, p.Scope)
 
 	objs, err = validate.Unstructured(ctx, p.Client, objs, options)
 
+	if !p.WebhookEnabled {
+		for _, obj := range objs {
+			klog.Infof("Removing %s annotation as Admission Webhook is disabled", metadata.DeclaredFieldsKey)
```

Comment:
You may want to move this log outside of the for loop to avoid spamming the logs.

Reply:
Done